fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneLockFailureTest.java

```diff
@@ -148,7 +152,12 @@ void testDeleteAll(boolean partitioned) throws IOException {
             grabLockExternally(partitioned, context, 2, 0);
             // Delete all just deletes the index, not through Lucene
             recordStore.deleteAllRecords();
-            context.commit();
+            try {
```

Comment:
I think we actually want this test to behave more like `testDeleteWhere` in that it should get the lock, then commit, then do `deleteAllRecords` in a separate context. I can followup with a quick PR to change that.